firestore-translate-text/__tests__/functions.test.ts

```diff
@@ -0,0 +1,384 @@
+import mockedEnv from "mocked-env";
+import * as functionsTestInit from "firebase-functions-test";
+
+const testTranslations = {
+  de: "hallo",
+  en: "hello",
+  es: "hola",
+  fr: "salut",
+};
+
+const defaultEnvironment = {
+  PROJECT_ID: "extensions-testing",
```

Comment:
is this a real project?

Reply:
It is a real project that we have - but in this instance it's just mock testing so can be any value here.

Reply:
can you revise to "fake-project" or something to make it obvious it is not using a real project?

Reply:
Agreed.